src/annotator/plugin/test/pdf-test.js

```diff
@@ -0,0 +1,121 @@
+import PDF, { $imports } from '../pdf';
+
+import FakePDFViewerApplication from '../../anchoring/test/fake-pdf-viewer-application';
+
+function delay(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('annotator/plugin/pdf', () => {
+  let container;
+  let fakeAnnotator;
+  let fakePDFViewerApplication;
+  let pdfPlugin;
+
+  function createPDFPlugin() {
+    const plugin = new PDF(document.body);
+    plugin.annotator = fakeAnnotator;
+    plugin.pluginInit();
+    return plugin;
+  }
+
+  beforeEach(() => {
+    // Setup fake PDF.js viewer.
```

Comment:
This comment is so simple and yet so potentially helpful down the road! Do you think it's potentially worth extracting this fake PDFjs setup to a test util? I guess maybe wait until we need to reuse it...?

Reply:
I think that would be best. I'm not sure how much of this setup will be universal yet.